src/algebra/char_p.lean

```diff
@@ -101,6 +109,14 @@ begin
   apply commute.all,
 end
 
+theorem sub_pow_char (α : Type u) [comm_ring α] {p : ℕ} (hp : nat.prime p)
```

Comment:
I think these versions could also use `[fact p.prime]`. I don't think `add_pow_char` is used *that* much, so if you want, please refactor it (-;